### PR TITLE
fix(NcModal): prev/next color and sizing fix

### PR DESCRIPTION
**Pull request by skjnldsv**
| Before | After |
|:---------:|:------:|
|||

### CODE REVIEW

File: src/components/NcModal/NcModal.vue [hunk 1]
```diff
@@ -882,11 +882,13 @@ export default {
...
 		display: flex !important;
-		height: 35vw;
+		height: 35vh;
+		min-height: 300px;
 		position: absolute;
 		transition: opacity 250ms,
 			visibility 250ms;
-		color: var(--color-primary-element-text);
+		// hover the mask
+		color: white;
```
Comment (szaimen) [inline src/components/NcModal/NcModal.vue:891]: Is this mask always black?

**Review by szaimen — APPROVED** [review]
Thanks!

**Review by susnux — APPROVED** [review]

**Comment by susnux** [comment]
BTW we need to fix the the actions and title, because it is lacking contrast

File: src/components/NcModal/NcModal.vue (hunk 1 above)
Comment (skjnldsv) [reply src/components/NcModal/NcModal.vue:891]: yep :+1: 
https://github.com/nextcloud-libraries/nextcloud-vue/blob/0e7db4c6a6518260396ebf47d6c786cd226e317b/src/components/NcModal/NcModal.vue#L743-L746

Even without dark mode, this is still black (with higher transparency, but still black)

**Pull request merged**